### DISCUSSION TITLE
various standard Groovy string methods give "no such property: <property> for class java.lang.String" error

### DISCUSSION THREAD

**Discussion by timboxyz** [freeplane/freeplane#1971] [Help]
I'm trying to manipulate various node text strings, but simple methods like `substring` & `getAt` and others produce the above error.

I'm assuming therefore that I may need to import a library although scripts in the samples section of Freeplane seem to use such methods with no import line so what am I missing.

I've tried both JAVA 11 and 21 in FP 1.11.14 on linux with the same result.

**Comment by euu2021** [comment]
Please, give an example

**Reply by timboxyz** [reply]
Well I was trying something like:-

```
def colourB =  "1-Blue"
c.statusInfo = colourB.getAt(2)
```
and getting the reported error. However, now it seems to be working so I don't know what is going on.
